ajax/libs/country-select-js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "country-select-js",
+  "filename": "build/css/countrySelect.min.css",
```

Comment:
I think the main files should be `countrySelect.min.js`. Refer https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/intl-tel-input/package.json#L3. It's the base library, which is hosted in cdnjs , of this library.

Besides this, filename the path to file should be relative path to `basePath` in auto-update config, so here would be `js/countrySelect.min.js`.